dataduct/pipeline/s3_node.py

```diff
@@ -59,6 +60,11 @@ def __init__(self,
         else:
             additional_args['filePath'] = s3_object
 
+        if (compression == "gzip"):
```

Comment:
I'm no longer one of the maintainers, but I'll offer my comments.

Nit: Please remove brackets around these statements, same for below.

Also, why are there no bzip2 and lzo options here?